pkg/scheduler/internal/cache/cache_test.go

```diff
@@ -367,12 +374,12 @@ func TestAddPodWillConfirm(t *testing.T) {
 		makeBasePod(t, nodeName, "test-1", "100m", "500", "", []v1.ContainerPort{{HostIP: "127.0.0.1", HostPort: 80, Protocol: "TCP"}}),
 		makeBasePod(t, nodeName, "test-2", "200m", "1Ki", "", []v1.ContainerPort{{HostIP: "127.0.0.1", HostPort: 8080, Protocol: "TCP"}}),
 	}
-	tests := []struct {
+	test := struct {
```

Comment:
Well, I'd like to keep the table driven test for the future extendability.

Reply:
I know, but keeping test tables conflicts with this comment https://github.com/kubernetes/kubernetes/pull/116849#discussion_r1206954663.
> In a follow up, we can remove the test table, as there is only one test. Or add more tests and a name.

 Many tests have only had one test since they were created, several years ago, but also have used test table.

If the test cases really need to be extended, is it better to add them later?